From ee2adbbfe4ff280b0c0378e14625d10c445fa0db Mon Sep 17 00:00:00 2001
From: Steven Koo <steven.koo@ni.com>
Date: Tue, 14 Jul 2020 18:35:03 +0000
Subject: [PATCH] Remove CONFIG_SYS_REDUNDAND_ENVIRONMENT check

We don't support u-boot v2020.01 yet so remove this check.
When we do, reevalute this patch and see if we can remove it.
See mender commit:
https://github.com/mendersoftware/meta-mender/commit/e60b35006a82fc5e108b5ef974e485250f6fe462

Signed-off-by: Steven Koo <steven.koo@ni.com>

---
 include/config_mender.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/include/config_mender.h b/include/config_mender.h
index c6a0eb2c23..99d4db6542 100644
--- a/include/config_mender.h
+++ b/include/config_mender.h
@@ -37,10 +37,6 @@
 # error CONFIG_BOOTCOUNT_ENV is required for Mender to work. Make sure that: 1) All the instructions at docs.mender.io/devices/integrating-with-u-boot have been followed. 2) All required layers are included in bblayers.conf, including any board specific layers such as meta-mender-<board>
 #endif
 
-#ifndef CONFIG_SYS_REDUNDAND_ENVIRONMENT
-# error CONFIG_SYS_REDUNDAND_ENVIRONMENT is required for Mender to work. Make sure that: 1) All the instructions at docs.mender.io/devices/integrating-with-u-boot have been followed. 2) All required layers are included in bblayers.conf, including any board specific layers such as meta-mender-<board>. Check also https://docs.mender.io/troubleshooting/yocto-project-build for Known Issues when upgrading.
-#endif
-
 #ifdef MENDER_UBI
 # ifndef CONFIG_MTDIDS_DEFAULT
 #  define CONFIG_MTDIDS_DEFAULT MENDER_MTDIDS
